assets/src/edit-story/utils/validateMinMax.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const validateMinMax = (value, { MIN, MAX }) => value >= MIN && value <= MAX;
```

Comment:
That's also a bit unclear naming, perhaps `isValueWithinLimits` or something similar would be more accurate, although not sure if this util is necessary at all.

Reply:
Maybe `inClamp` to match `clamp`? `inRange/toRange` is also possible.

Reply:
`inClamp` or `inRange` -- either of these is good 👍